packages/example/pages/index.tsx

```diff
@@ -70,9 +70,15 @@ export default function Home(): React.ReactElement {
       kit.contracts.getGoldToken(),
       Promise.all(
         Object.values(StableToken).map(async (stable) => {
+          let contract;
+          try {
+            contract = await kit.contracts.getStableToken(stable);
+          } catch (e) {
```

Comment:
since it's the example app it's fine, but maybe we could log the error here just in case

Reply:
done!